autoload/neomake/makers/ft/python.vim

```diff
@@ -83,6 +83,11 @@ function! neomake#makers#ft#python#Flake8EntryProcess(entry) abort
     else
         let type = ''
     endif
+    let l:token = matchstr(a:entry.text, "'.*'")
+    if strlen(l:token)
+        let a:entry.length = strlen(l:token) - 2 " subtract the quotes
+    endif
```

Comment:
@paretje: that's an extension of @JelteF to have highlights of variable lengths when makers produce such information. What should be separated? The implementation for python, or the whole feature?